libs/shared/methodologies/bold/helpers/src/homologation-document.helpers.ts

```diff
@@ -0,0 +1,57 @@
+import { isNil } from '@carrot-fndn/shared/helpers';
+import { getEventAttributeValue } from '@carrot-fndn/shared/methodologies/bold/getters';
+import {
+  eventNameIsAnyOf,
+  isOpenEvent,
+} from '@carrot-fndn/shared/methodologies/bold/predicates';
+import {
+  type Document,
+  DocumentEventAttributeName,
+  DocumentEventName,
+} from '@carrot-fndn/shared/methodologies/bold/types';
+import { type NonEmptyString } from '@carrot-fndn/shared/types';
+import { format } from 'date-fns';
+import { is } from 'typia';
+
+export const getParticipantHomologationDocumentByParticipantId = ({
+  homologationDocuments,
+  participantId,
+}: {
+  homologationDocuments: Document[];
+  participantId: NonEmptyString;
+}): Document | undefined =>
+  homologationDocuments.find((document) => {
+    const openEvent = document.externalEvents?.find(isOpenEvent);
+
+    return !isNil(openEvent) && openEvent.participant.id === participantId;
+  });
+
+export const isHomologationInForce = (document: Document): boolean => {
```

Comment:
Even though the expression `in force` is correct, I believe that `is homologation active` express better the period validation we are doing here, and it clearer for the reader.
```suggestion
export const isHomologationActive = (document: Document): boolean => {
```

Reply:
https://github.com/carrot-foundation/methodology-rules/pull/130/commits/45de98f043257724f207e274321325b32268abfb